### ISSUE TITLE
L410, YK40, SU95, AN2, IL76

### ISSUE THREAD

**Issue by j14a** [VATSIM-Radar/vatsim-radar#404]
### Your Vatsim CID

1682195

### What aircraft icon(s) do you suggest to add

Let L-410 Turbolet - L410
Yakovlev Yak-40 - YK40
Sukhoi Superjet 100-95 - SU95
Antonov An-2 - AN2
Ilyushin Il-76 - IL76
![ac-above](https://github.com/user-attachments/assets/fbe4a83b-23db-4940-9970-a44b007a66d3)

### Validation

- [X] I have searched for other requests and didn't found any alike

**Comment by KuezyIT** [comment]
Something I noticed was that when I flew the Panavia Tornado yesterday on VATSIM, it didn't show up as a jet on vatsim-radar. Maybe this could also be implemented? It doesn't need its own icon, just the one that is e.g., used for the Eurofighter.

**Comment by DotWallop** [comment]
All added. Also added TOR from @KuezyIT's suggestion. :)

**Comment by KuezyIT** [comment]
Has the Icon for the Tornado already been implemented?:
![image](https://github.com/user-attachments/assets/e3c8a754-fc8b-4512-949f-f8fee4171016)

**Comment by DotWallop** [comment]
No. 😀 When the icons are implemented, the topic will be closed with a "completed" label.

**Comment by KuezyIT** [comment]
Sounds good, thanks!
